wiki/Project_Loved/nl.md

```diff
@@ -0,0 +1,42 @@
+# Project Loved
```

Comment:
outdated translation

Reply:
This one is the same as master, I can't find the new version?

Reply:
your translation has two `Meer informatie` headers and 6 headers in general, while the English version has 5 headers

Reply:
Sorry, no idea what happened there. Fixed!